configuration/pih/scripts/global/zl.js

```diff
@@ -0,0 +1,144 @@
+function setUpEdd(currentEncounterDate, msgWeeks) {
+  var updateEdd = function () {
+    var lastPeriodDate = htmlForm.getValueIfLegal("lastPeriodDate.value");
+    if (
+      typeof lastPeriodDate !== "undefined" &&
+      lastPeriodDate !== null &&
+      lastPeriodDate.length > 0
+    ) {
+      var today = new Date();
+      if (
+        typeof currentEncounterDate !== "undefined" &&
+        currentEncounterDate !== null &&
+        currentEncounterDate.length > 0
+      ) {
+        // calculate the gestational age at the time of the encounter
+        today = new Date(+currentEncounterDate);
+      }
+
+      var dateObj = getField("lastPeriodDate.value").datepicker("getDate");
+      var newDate = new Date(dateObj);
+      // time difference
+      var timeDiff = Math.abs(today.getTime() - newDate.getTime());
+      // weeks difference = gestational age
+      var diffWeeks = Math.ceil(timeDiff / (1000 * 3600 * 24 * 7));
+
+      // Estimated Delivery Date = (LMP - 3 months) + 12 months + 7 days
+      newDate.setMonth(newDate.getMonth() - 3);
+      newDate.setFullYear(newDate.getFullYear() + 1);
+      newDate.setDate(newDate.getDate() + 7);
+
+      var widgetDate = getField("lastPeriodDate.value")
+        .datepicker("setDate", newDate)
+        .val();
+      getField("lastPeriodDate.value").datepicker("setDate", dateObj);
+
+      jq("#calculated-edd-and-gestational").show();
+      jq("#calculated-edd").text(widgetDate);
+      jq("#calculated-gestational-age-value").text(diffWeeks + " " + msgWeeks);
+    } else {
+      jq("#calculated-edd-and-gestational").hide();
+    }
+  };
+
+  jq("#calculated-edd-and-gestational").hide();
+
+  jq("#lastPeriodDate input[type='hidden']").change(function () {
+    updateEdd();
+  });
+
+  updateEdd();
+}
+
+function setUpNextButtonForSections() {
+  // handle our custom functionality for triggering going to the next section when the "Next" button is clicked
+  jq("#next").click(function () {
+    window.htmlForm.getBeforeSubmit().push(function () {
+      window.htmlForm.setReturnUrl(
+        window.htmlForm.getReturnUrl() + "&amp;goToNextSection=pmtct-history"
+      );
+      return true;
+    });
+
+    window.htmlForm.submitHtmlForm();
+  });
+
+  jq("#submit").click(function () {
+    window.htmlForm.submitHtmlForm();
+  });
+}
+
+function setUpExpandableContacts() {
+  var x = 1;
```

Comment:
```suggestion
  let numberToShow = 1;
```

Don't use single-letter variable names.